docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
ZSlDe+/SmeTUqnf7OXqXLSBJTed1h7He6Lz2++1fAjJ6CU2WUI+NavT9D++mWcbELCGyxPMJtzWRbNVjAYulqloa4kyeYSxJLB8qjPpwYo6dzFOrRs6wWYhxNvLMNJp5bXlAtq+9qsRkLaPX4cS+oFpqlfv7co8+7R4XF+2NPbRJg5wfs6j6Jpj59VIGxmfQrfuZ6iOC5cCghEToyvcA5AQ3B6Y+UvY4lAjKs9Iwb3CTYKRVhOQzbT+L6pduCYnf7+ittHxq/KNgRswp3Rv5KuCrXDzZm44qHTiBq7y2KXR8qMe7drA3cJnyQGDddsx0NtoVazjcwA5YQatK1tKmpi2TTtm+gs3Cef73Y5ERSBIg/uZU0YzmTZhVxVtAWGiHkgoWy7A6kLydeUuHYhOUO5TNz4/uGIaIcIqNcVM048Y16svBKjdECZCuFHQbYmEAnhyQQvMB7MrhjJwTduzmw1CdWcSwnXDYAMBSestWAeEaXjMgiEIbUkQE4J48PIcvcQmEspv5QEcC4LEJ6fQXI4DYVZKCwqrBP4KbWy4aCs27GCgS1G1KpXpjVf6aMnZfhEHQryn37jJ1LVNC2NAedTAntnyPa87MONnDemsOincAbhKmJnRprjh6mD2SRMQBqyZa7fv0cA5Kyjxxi0V00HsITdpU+/JzAim30Es0K/FSMyV04gJbhSCr2gHyuVHEOuAeLA7roknnycEBGkqpm0zPY5txLBem/cbPeK/xjWrcbzLYEHlzIf9+o+J1OXxS9lN/7HJoNAA+mLdPoUrLBlIvxDzlr5KNPnScMAhxIAgeeFQ3+MJqZuWgcv4Bya0ReGnqC9bransCQDfYRyn01uwEBH9BNPwmAWzJLtORpEXe0K1HrtllvLmN82uZkDDej40nNf4YMXz5BlfU3kXZ5r05rBsbcO0OQLPNzhCP2fRcoqOvZwt3XTno7dGqOA1gskLf12v3DMcGzXg+SehWA4qI/AKQcB5YHN/tR2VkGIcldFxV4wfQHB7tA8Cr4g/gIAa1qhCqya+TP/Lbg9uVnOiWSpIc+frhf17/PpZrAnSRGuHtq/Fq7R9a36xzAACa06TqMfbNSs/sdCQZto2La/2i0Ax4jXDWwbRD2/0pldaBGpJT/1a+tc0fUPq8WBYksIvxCtXDJaRFmBGrA0C19Cc2IKDpTJBPLmAAaCln5eKSe2dMpx685IMvQUrobEQETNkIJnc5IvFK+kMT+N3C22SWxabjltlCSGAe+3IBosVxe76JVv6/02ZSH9j1w1+p5OjYXT44PUhxjzoWPM/+4ju3L1HJfniay7TnE/eJ3xyyicTRhM+qRxMc3PN3knfGtKf8Mkpjer7PtXuWR+/S/hoArcwr3PT8hbG79ccqwGZIHDpxB1zB68XgjSvU12RSxATTATPtoyZv446BXjTTQ5ihl+4TIs4yIkZLQjcr5/2cUfjQKi2Tkcl5PKFm65LwzoEF2/M/N+oB7sl0OFT1ZXe7RKaSynrDsO6QdxjU2QMvb1wI6XxsorTianKVbNk+eSsxe/xOR6LS1LWw/JDBjcvZ7689jFT1pnPeR0HqCF9TawgWDpwc88kXlXuMMj4I44B2UsQs8KAm5GZRReuXFNudOt6NXjQsewy2dIerITGQkSsA4LLhVmwmmzt4ByxD3KKTTIMFAm8RCbE74mbbEt9pmWsIiHTgtZsY0YsL3Eum/8Vt9CVk4Dy0l/c00E+MTytuzUqVsUImvE+4iWIx0BmD+3KQeE9atp+H5MPBNO9ENw8L3PQdS3HkKav3cuzar5qBuirLYdVuqUvBFrvP3io9MI1wMiMeIVegjjTvNE91cwelhzgpoOl9ynHpTQ3Mz91IZi+g11RBHpP8zT9OzxDd5DdqXMc0LVWBjNWICG4P33YMtIEVsub7UPo3zwD9j1/TN0vLY9ev/Bmw3iJAab7EaA2lWVv91cDj+kxPbaxEhoLEbEOCNPwpgIXbHTPjgQCWn/AMMvaK6yZaRabt616dszmWqSgRQSseXYMwWItTmNOKUYuzFZVdC5DTjn+IybcKjxak/O4OT05j+FRC8SIKrFexA/sIVi3w82zinxytgMxpnIc6wGB2CoI/E/8cbOaMZv3Ws+n1MAdu8tr5bBMwl4CosMU/ge8NKBHY1/1H3kRsMfShlFtGqnj9qoberEOSD3Ayp4KgtCmMaC1+w4rIureIpgXOD5NRHZ9dsyeJy2684eApRgCNMC9hXVhTqyxLHKWB/HknO3Kmaeg+3B1CiqYxbJVjvIix0MxiezORai40B8fr7HKqQ9uqj0JCPWdYZcU4ljKQRFQAsf0sAgTYk8LTFXAX1gDlNs6II14rvBsmh4ilpAeKBZQhzWWUAGrkuIWEPjH+gM0l9dWYTnEhM7NjVPnForYLQkrqvG8GVaAaxMBWGg5olUjeqtDiN3q1UOxtZaQkJAeqBdQh5BQAQHUqTjcWKxvQiC/IO6cTs1RauBCY90S1nnhQWVtvZCedeitS0hIb0SfgT4hoWcAslTxzBCwloTsE2F8ZqcrFQkCgkBxR0BIqLhfQWm/IFDMERASKuYXUJovCBR3BISEivsVlPYLAsUcASGhYn4BpfmCQHFHQEioCFxBPFcnIgg8DwjY0peFhIrAlXfk57FEBIHnAQFb+rKQUBG48s5lDP8tpAg0RZogCBQIAVv6spBQgSDXpzDeb1PBxbq3COhTs2gRBPRDAH3Y0ruacqtJ/IDckHnGx8vyk994SPTOvQeUwUv1s3J5cdYzbpZUJwhYRAAxILhgsIBsISAol2fHLEIsiYKAIGBvBMQdszfCol8QEAQsIiAkZBEeSRQEBAF7IyAkZG+ERb8gIAhYREBIyCI8kigICAL2RkBIyN4Ii35BQBCwiICQkEV4JFEQEATsjYCQkL0RFv2CgCBgEQEhIYvwSKIgIAjYGwEhIXsjLPoFAUHAIgJCQhbhkURBQBCwNwL/AedZ9yFiupMfAAAAAElFTkSuQmCC"
```

Comment:
I think it would be nice if you added a mini explanation of what the image was.

Reply:
+1, for example https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/amazon_textract_connector_blueprint.md#step-5-test-the-model-inference